PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,12 +14,10 @@ What features does this change enable? What bugs does this change fix?
 
 <!--
 
-Please provide a list of JIRA tickets that this Pull Request acts upon. This section will be parsed, 
-a comment will be added, and the FixVersion field of the JIRA will be set to DC/OS release version upon merged.
-The FixVersion will correspond to the [fix_version_map](https://github.com/dcos/dcos/blob/master/mergebot-config.json#L30) 
-of the mergebot-config.
+Please provide a list of JIRA tickets that this Pull Request acts upon. This section will be parsed, a comment will be added, and the FixVersion field of the JIRA will be set to DC/OS release version upon merged.
```

Comment:
This is great. I have tried to re-order this a little bit to make it hopefully even easier to read! Proposal:

"""
This section will be parsed for JIRA tickets. Please provide a list of JIRA tickets that this Pull Request is supposed to act upon. Upon merge, every JIRA ticket listed here will be modified by Mergebot:
- a comment will be added
- the `fixVersion` field of the JIRA ticket will be set to the currently configured DC/OS release version.
"""

Reply:
I have adopted this with a slight tweak. I have added this to the header section of the PR and presented the current parsing behavior so that it is clear to all the users and is presented upfront. Thank you!